src/main.js

```diff
@@ -1,8 +1,10 @@
-import Application from './Application.vue';
+import 'core-js/stable';
```

Comment:
what are these two for?

Reply:
These two are needed to convert `.vue` files to es5.
According to this article https://babeljs.io/docs/en/babel-polyfill

Reply:
doesn't the webpack build set up already do so? I believe both @tonytlwu and Hugo have written components that transpile to ES5 without needing these files.

Reply:
@hcarneiro configured webpack so he can confirm. @YaroslavOvdii can you confirm if ES5 transpiling is already happening without your additions?

Reply:
@tonytlwu Yes, they are happening but when we tried out this widget on IE11 without this lines we encounter these errors:
![image](https://user-images.githubusercontent.com/53430352/82925495-6eb4d400-9f86-11ea-806e-f816e39a564e.png)
It seems that without these lines webpack doesn't create polyfills for IE.
In this particular case we polyfills `Object.keys` and `Object.assing` features from es6.
Lines we have add do this for us.